Haskell-Generate/GenRustJets.hs

```diff
@@ -234,6 +236,24 @@ rustJetDecode mod =
                      <-> ("1" <+> "=>" <+> braces (docTree r))
                        ) <> line
 
+rustJetCost :: Module -> Doc a
+rustJetCost mod = vsep $
+  [ nest 4 (vsep ("fn cost(&self) -> Cost {" :
+    [ nest 4 (vsep ("match self {" :
+        map (<>comma)
+        [ pretty modname <> "::" <> pretty (jetName jet) <+> "=>" <+>
+          if BitcoinModule == jetModule jet
+          then "unimplemented!(\"Unspecified cost of Bitcoin jets\")"
+          else "Cost::from_jet(" <> (pretty $ milliWeight $ cost $ jetName jet) <> ")"
```

Comment:
I believe the usual idiom in Haskell is to write

    pretty . milliWeight . cost $ jetName jet

Reply:
Function composition, nice. I pushed an update.